### PR TITLE
breaking: bump seed, scala, sbt, play-json

### PR DESCRIPTION
**Pull request by pedrocodacy**

### CODE REVIEW

**Comment by pedrocodacy** [comment]
I found that the two tests that were not passing had test files in LESS and not CSS.
I checked our definition for the language that this tool analyses (CSS) in codacy-plugins-api and found that it only maps to .css files.

Since there were already .css files that tested the exact same patterns I removed the .less files and now the tests pass. 

Do you guys think I should keep trying to get node 7.10.1 to work on this image, or does this seem like the right thing to do?

**Comment by prendi** [comment]
If the LESS files are really not needed I think we shouldn't waste more time with it. In any case try to figure out why we had tests with .less files. Maybe @rtfpessoa ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            - "~/codacy-plugins-test/target"
+            - "~/codacy-plugins-test/project/target"
+            - "~/codacy-plugins-test/project/project"
+            - "~/workdir/target"
+            - "~/workdir/project/target"
+            - "~/workdir/project/project"
+      - run:
+          name: Test json
+          working_directory: ~/codacy-plugins-test
+          command: sbt -Dcodacy.tests.ignore.descriptions=true "runMain codacy.plugins.DockerTest json $CIRCLE_PROJECT_REPONAME:latest"
```
Comment (prendi) [inline .circleci/config.yml]: don't forget the codacy/

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      - run:
+          name: Test patterns
+          working_directory: ~/codacy-plugins-test
+          command: sbt -Dcodacy.tests.noremove=true -Dcodacy.tests.threads=8 "runMain codacy.plugins.DockerTest pattern $CIRCLE_PROJECT_REPONAME:latest"
```
Comment (prendi) [inline .circleci/config.yml]: don't forget the codacy/

File: build.sbt [hunk 3]
```diff
@@ -48,15 +51,14 @@ daemonUser in Docker := dockerUser
 
 daemonGroup in Docker := dockerGroup
 
-dockerBaseImage := "frolvlad/alpine-oraclejdk8"
+dockerBaseImage := "frolvlad/alpine-java"
```
Comment (DReigada) [inline build.sbt]: We should use a fixed tag for the image.
Either `jre8.202.08-slim` or `jre8.202.08-cleaned`

**Pull request merged**